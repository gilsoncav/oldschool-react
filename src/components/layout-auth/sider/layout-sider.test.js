import LayoutSider from './layout-sider';

describe('LayoutSider component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<LayoutSider />);
    expect(wrapper).toMatchSnapshot();
  });
});
