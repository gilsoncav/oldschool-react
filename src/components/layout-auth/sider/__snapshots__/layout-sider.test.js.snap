// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutSider component renders correctly collapsed 1`] = `
<aside
  className="ant-layout-sider ant-layout-sider-dark ant-layout-sider-collapsed ant-layout-sider-zero-width"
  style={
    Object {
      "flex": "0 0 0px",
      "maxWidth": "0px",
      "minWidth": "0px",
      "width": "0px",
    }
  }
>
  <div
    className="ant-layout-sider-children"
  />
</aside>
`;

exports[`LayoutSider component renders correctly uncollapsed 1`] = `
<aside
  className="ant-layout-sider ant-layout-sider-dark"
  style={
    Object {
      "flex": "0 0 200px",
      "maxWidth": "200px",
      "minWidth": "200px",
      "width": "200px",
    }
  }
>
  <div
    className="ant-layout-sider-children"
  >
    <ul
      className="ant-menu ant-menu-dark ant-menu-root ant-menu-inline"
      id="sider-menu"
      onMouseEnter={[Function]}
      onTransitionEnd={[Function]}
      role="menu"
      style={Object {}}
    >
      <li
        className="ant-menu-item ant-menu-item-selected"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        role="menuitem"
        style={
          Object {
            "paddingLeft": 24,
          }
        }
      >
        <i
          aria-label="icon: home"
          className="anticon anticon-home"
        >
          <svg
            aria-hidden="true"
            className=""
            data-icon="home"
            fill="currentColor"
            focusable="false"
            height="1em"
            key="svg-home"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M946.5 505L534.6 93.4a31.93 31.93 0 0 0-45.2 0L77.5 505c-12 12-18.8 28.3-18.8 45.3 0 35.3 28.7 64 64 64h43.4V908c0 17.7 14.3 32 32 32H448V716h112v224h265.9c17.7 0 32-14.3 32-32V614.3h43.4c17 0 33.3-6.7 45.3-18.8 24.9-25 24.9-65.5-.1-90.5z"
              key="svg-home-svg-0"
            />
          </svg>
        </i>
        <span>
          Home
        </span>
      </li>
      <li
        className="ant-menu-item"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        role="menuitem"
        style={
          Object {
            "paddingLeft": 24,
          }
        }
      >
        <i
          aria-label="icon: file-search"
          className="anticon anticon-file-search"
        >
          <svg
            aria-hidden="true"
            className=""
            data-icon="file-search"
            fill="currentColor"
            focusable="false"
            height="1em"
            key="svg-file-search"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M688 312v-48c0-4.4-3.6-8-8-8H296c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8h384c4.4 0 8-3.6 8-8zm-392 88c-4.4 0-8 3.6-8 8v48c0 4.4 3.6 8 8 8h184c4.4 0 8-3.6 8-8v-48c0-4.4-3.6-8-8-8H296zm144 452H208V148h560v344c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8V108c0-17.7-14.3-32-32-32H168c-17.7 0-32 14.3-32 32v784c0 17.7 14.3 32 32 32h272c4.4 0 8-3.6 8-8v-56c0-4.4-3.6-8-8-8zm445.7 51.5l-93.3-93.3C814.7 780.7 828 743.9 828 704c0-97.2-78.8-176-176-176s-176 78.8-176 176 78.8 176 176 176c35.8 0 69-10.7 96.8-29l94.7 94.7c1.6 1.6 3.6 2.3 5.6 2.3s4.1-.8 5.6-2.3l31-31a7.9 7.9 0 0 0 0-11.2zM652 816c-61.9 0-112-50.1-112-112s50.1-112 112-112 112 50.1 112 112-50.1 112-112 112z"
              key="svg-file-search-svg-0"
            />
          </svg>
        </i>
        <span>
          Library
        </span>
      </li>
      <li
        className="ant-menu-item"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        role="menuitem"
        style={
          Object {
            "paddingLeft": 24,
          }
        }
      >
        <i
          aria-label="icon: star"
          className="anticon anticon-star"
        >
          <svg
            aria-hidden="true"
            className=""
            data-icon="star"
            fill="currentColor"
            focusable="false"
            height="1em"
            key="svg-star"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M908.1 353.1l-253.9-36.9L540.7 86.1c-3.1-6.3-8.2-11.4-14.5-14.5-15.8-7.8-35-1.3-42.9 14.5L369.8 316.2l-253.9 36.9c-7 1-13.4 4.3-18.3 9.3a32.05 32.05 0 0 0 .6 45.3l183.7 179.1-43.4 252.9a31.95 31.95 0 0 0 46.4 33.7L512 754l227.1 119.4c6.2 3.3 13.4 4.4 20.3 3.2 17.4-3 29.1-19.5 26.1-36.9l-43.4-252.9 183.7-179.1c5-4.9 8.3-11.3 9.3-18.3 2.7-17.5-9.5-33.7-27-36.3z"
              key="svg-star-svg-0"
            />
          </svg>
        </i>
        <span>
          Your List
        </span>
      </li>
      <li
        className="ant-menu-item"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        role="menuitem"
        style={
          Object {
            "paddingLeft": 24,
          }
        }
      >
        <i
          aria-label="icon: info-circle"
          className="anticon anticon-info-circle"
        >
          <svg
            aria-hidden="true"
            className=""
            data-icon="info-circle"
            fill="currentColor"
            focusable="false"
            height="1em"
            key="svg-info-circle"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm32 664c0 4.4-3.6 8-8 8h-48c-4.4 0-8-3.6-8-8V456c0-4.4 3.6-8 8-8h48c4.4 0 8 3.6 8 8v272zm-32-344a48.01 48.01 0 0 1 0-96 48.01 48.01 0 0 1 0 96z"
              key="svg-info-circle-svg-0"
            />
          </svg>
        </i>
        <span>
          About
        </span>
      </li>
    </ul>
  </div>
</aside>
`;
